src/main/java/com/iota/iri/conf/BaseIotaConfig.java

```diff
@@ -745,6 +759,9 @@ protected void setPowThreads(int powThreads) {
         //TipSel
         int MAX_DEPTH = 15;
         double ALPHA = 0.001d;
+        
+        //Tip solidification
+        boolean TIP_SOLIDIFIER_ENABLED = true;
```

Comment:
@jakubcech 
When you wrote the requirements why did you decide that it should be true?
IIRC we decided this thread consumes resources and might cause crashes and better be turned off.

Reply:
Thinking was that people can turn it off if they experience problems. I mentioned that after submitting. I'm for making it false by default if it makes sense (pending investigation iiuc).

Reply:
@hmoog 
Said it works fine for him

However if we don't have enough time to test for the change (community testers) we can leave it true and give a tip to the community to turn it off

Reply:
^ Agree